Makefile

```diff
@@ -2,6 +2,8 @@ DATE		= $(shell date)
 PYTHON		= $(shell which python)
 PKG_VERSION = $(shell poetry -s version)
 BUILD_DATE  = $(shell date +'%B %d, %Y')
+QPC_VAR_PKG_SOURCE := $(or $(QPC_VAR_PKG_SOURCE), "qpc")
```

Comment:
Where did the name `QPC_VAR_PKG_SOURCE` come from?

At a glance, I wouldn't know what `QPC_VAR_PKG_SOURCE` means if I'm not already familiar with this code. I understand `QPC_` is a prefix we want to standardize, but `VAR_PKG_SOURCE` doesn't have an intrinsic meaning to me. It may be easy to understand if you find the original definition _here_, but I wonder if a name with a more direct meaning would make it more readable in the docs and code. Maybe a name like `QPC_PROGRAM_NAME` is more precise assuming it is always meant to refer to the program `qpc` or `dsc`.

Thoughts?

Reply:
PKG_SOURCE was the original name, and I didnt think about changing it, Ive just added a prefix to be able to use a regex when importing env variables. QPC_VAR_PROGRAM_NAME sounds good, I will make the changes.